src/fromevent.js

```diff
@@ -1,8 +1,7 @@
-let Rx = require(`rx`)
+let Rx = require(`@reactivex/rxjs`)
 
-const disposableCreate = Rx.Disposable.create
-const CompositeDisposable = Rx.CompositeDisposable
-const AnonymousObservable = Rx.AnonymousObservable
+const disposableCreate = (func) => new Rx.Subscription(func)
```

Comment:
This was just quick fixes to get up and running with RxNext. If it’s used frequently, it is also good for code minimization. However, I don’t think that it is.

The whole library is in need of a good refactoring, IMO. So I just deferred it.